test/models/userAnswersToEtmp/Reads/CommonGenerator.scala

```diff
@@ -0,0 +1,296 @@
+      emailAddress <- arbitrary[String]
+    } yield Json.obj(
+      "trusteeDetails" -> Json.obj(
+        "firstName" -> firstName,
+        "lastName" -> lastName,
+        "isDeleted" -> isDeleted
+      ),
+      "dateOfBirth" -> dateOfBirth,
+      "trusteeAddressId" -> correspondenceAddressDetails,
+      "trusteeAddressYears" -> addressYears,
+      "trusteePreviousAddress" -> previousAddressDetails,
+      "trusteeContactDetails" -> Json.obj(
+        "emailAddress" -> emailAddress,
+        "phoneNumber" -> mobileNumber
+      )
+    )
+
+  val trusteesGen: Gen[JsObject] =
+    for {
+      company1 <- trusteeCompanyGenerator()
+      company2 <- trusteeCompanyGenerator(true)
+      individual3 <- trusteeIndividualGenerator(true)
+      individual4 <- trusteeIndividualGenerator()
+      partnership5 <- trusteePartnershipGenerator()
+      partnership6 <- trusteePartnershipGenerator(true)
+    } yield {
+      Json.obj(
+        "trustees" -> Seq(
+          company1, company2, individual3, individual4, partnership5, partnership6
+        )
+      )
+    }
+
+  def directorGenerator(isDeleted: Boolean = false): Gen[JsObject] =
+    for {
+      firstName <- arbitrary[String]
+      lastName <- arbitrary[String]
+      dateOfBirth <- arbitrary[String]
+      correspondenceAddressDetails <- CommonGenerator.addressGen
+      addressYears <- arbitrary[String]
+      previousAddressDetails <- CommonGenerator.addressGen
+      mobileNumber <- arbitrary[String]
+      emailAddress <- arbitrary[String]
+    } yield Json.obj(
+      "directorDetails" -> Json.obj(
+        "firstName" -> firstName,
+        "lastName" -> lastName,
+        "isDeleted" -> isDeleted
+      ),
+      "dateOfBirth" -> dateOfBirth,
+      "directorAddressId" -> correspondenceAddressDetails,
+      "companyDirectorAddressYears" -> addressYears,
+      "previousAddress" -> previousAddressDetails,
+      "directorContactDetails" -> Json.obj(
+        "emailAddress" -> emailAddress,
+        "phoneNumber" -> mobileNumber
+      )
+    )
+
+  def partnerGenerator(isDeleted: Boolean = false): Gen[JsObject] =
+    for {
+      firstName <- arbitrary[String]
+      lastName <- arbitrary[String]
+      dateOfBirth <- arbitrary[String]
+      correspondenceAddressDetails <- CommonGenerator.addressGen
+      addressYears <- arbitrary[String]
+      previousAddressDetails <- CommonGenerator.addressGen
+      mobileNumber <- arbitrary[String]
+      emailAddress <- arbitrary[String]
+    } yield Json.obj(
+      "partnerDetails" -> Json.obj(
+        "firstName" -> firstName,
+        "lastName" -> lastName,
+        "isDeleted" -> isDeleted
+      ),
+      "dateOfBirth" -> dateOfBirth,
+      "partnerAddressId" -> correspondenceAddressDetails,
+      "partnerAddressYears" -> addressYears,
+      "partnerPreviousAddress" -> previousAddressDetails,
+      "partnerContactDetails" -> Json.obj(
+        "emailAddress" -> emailAddress,
+        "phoneNumber" -> mobileNumber
```

Comment:
The package name should start with lower case r, i.e reads, not Reads. Confused me a bit cause I was looking for a file called Reads.scala!